src/contents/all/components/app/LoginForm.tsx

```diff
@@ -0,0 +1,101 @@
+/*global chrome*/
+
+import React, { useEffect, useState } from 'react';
+import Frame, { FrameContextConsumer } from 'react-frame-component';
+import { useDispatch, useSelector } from "react-redux";
+import { Link } from 'react-router-dom';
+import { hot } from 'react-hot-loader/root';
+import * as Sentry from '@sentry/browser';
+
+import { Form, Icon, Input, Button, message, notification } from 'antd';
+
+import './App.scss';
+
+//import './LoginForm.scss'
+
+interface Props {
+  form: any
+}
+
+const FormItem = Form.Item;
+
+const LoginForm = () => {
+
+  const {error, inProgress} = useSelector(state => ({ ...state.auth, ...state.common}))
+
+  const submitForm = ev => {
+    ev.preventDefault();
+    form.validateFieldsAndScroll((err, values) => {
+      if (!err) {
+        chrome.runtime.sendMessage({
+          message: "auth_user",
+          user: {
+            email: values.email,
+            password: values.password
+          }
+        })
+      }
+    })
+  }
+
+  useEffect(()=>{
+    if (error) {
+      if (error instanceof String) {
+        message.error(error)
+      } else {
+        for (const [ key, value ] of Object.entries(error)) {
+          message.error(`${key.charAt(0).toUpperCase()}${key.slice(1)} ${value}`)
+        }
+      }
+    }
+  },[inProgress])
+
+  const { getFieldDecorator } = form;
+
+  return (
+    <div className="loginPage">
+      <div className="loginForm">
+        <Form onSubmit={submitForm}>
+          <FormItem>
+            {getFieldDecorator('email', {
+              rules: [
+                { required: true, message: 'Please enter your email' },
+                { type: 'email', message: 'Not a valid email' }
+              ],
+              validateTrigger: "onBlur"
+            })(
+              <Input
+                size="large"
+                prefix={<Icon type="mail" />}
+                placeholder="Email"
+              />
+            )}
+          </FormItem>
+          <FormItem>
+            {getFieldDecorator('password', {
+              rules: [
+                { required: true, message: 'Please enter your password' }
+              ],
+              validateTrigger: "onBlur"
+            })(
+              <Input.Password
+                size="large"
+                prefix={<Icon type="lock" />}
+                placeholder="Password"
+              />
+            )}
+          </FormItem>
+          <FormItem>
+            <Button htmlType="submit" loading={inProgress}>
+              Log in
+            </Button>
+          </FormItem>
+        </Form>
+      </div>
+      <p className="formFooter formLink"><Link to="/password/reset" className="">Forgot your password?</Link></p>
+      <p className="formFooter">New to Analogue? <Link to="/signup">Sign Up</Link></p>
+    </div>
+  )
+}
+
+export default LoginForm;
```

Comment:
@spencerjsmall In order to use ant designs `form` component and API, we would wrap this in a higher order component like so:
```suggestion
export default Form.create<Props>()(LoginForm)
```

See ant design docs for more details on Form: https://3x.ant.design/components/form/